### PR TITLE
Move mod/rsd_xml to src/Module/ReallySimpleDiscovery

### PR DESCRIPTION
**Pull request by nupplaphil**
Part of #4090

### CODE REVIEW

File: src/Module/ReallySimpleDiscovery.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Prints the rsd.xml
+ * @see http://danielberlinger.github.io/rsd/
+ */
+class ReallySimpleDiscovery extends BaseModule
+{
+	public static function rawContent()
+	{
+		header ("Content-Type: text/xml");
```
Comment (MrPetovan) [inline src/Module/ReallySimpleDiscovery.php]: Formatting: Please normalize the quote style in this file.

File: src/Module/ReallySimpleDiscovery.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		$tpl = Renderer::getMarkupTemplate('rsd.tpl');
+		echo Renderer::replaceMacros($tpl, []);
```
Comment (MrPetovan) [inline src/Module/ReallySimpleDiscovery.php]: You can ditch the `[]`, it is implied in the method prototype.

File: src/Module/ReallySimpleDiscovery.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * Prints the rsd.xml
+ * @see http://danielberlinger.github.io/rsd/
+ */
+class ReallySimpleDiscovery extends BaseModule
+{
+	public static function rawContent()
+	{
+		header ('Content-Type: text/xml');
+		$tpl = Renderer::getMarkupTemplate('rsd.tpl');
```
Comment (annando) [inline src/Module/ReallySimpleDiscovery.php]: It's a little bit late, but I would prefer having all XML templates replaced with creating XML via the XML routines.

Comment (nupplaphil) [reply src/Module/ReallySimpleDiscovery.php]: There's a XML routine :X .. Sorry I didn't notice yet

Comment (annando) [reply src/Module/ReallySimpleDiscovery.php]: Yeah, that routine can be feed with an array that is then transformed in an XML. Have a look at how ```XML::fromArray``` is used in the current code.

**Pull request merged**